server/command.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+const awsSNSCmd = "awssns"
+
+func (p *Plugin) registerCommands() error {
+	if err := p.API.RegisterCommand(&model.Command{
+		Trigger:          awsSNSCmd,
+		Description:      "Mattermost slash command to interact with AWS SNS",
+		DisplayName:      "AWS SNS",
+		AutoComplete:     true,
+		AutoCompleteHint: "listTopics",
+		AutoCompleteDesc: "List Topics which are subscribed to the channel",
+	}); err != nil {
```

Comment:
0/5 I would prefer initializing `err` and check if `nil` below, versus the inline `err != nil`